06-gitops.md

```diff
@@ -46,13 +50,38 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
+
+   ```bash
+   # Get your ACR cluster name
+   export ACR_NAME=$(az deployment group show --resource-group rg-bu0001a0008 -n cluster-stamp --query properties.outputs.containerRegistryName.value -o tsv)
+
+   # Import cluster management images hosted in public container registries
+   az acr import --source docker.io/library/memcached:1.5.20 -n $ACR_NAME
+   az acr import --source docker.io/fluxcd/flux:1.19.0 -n $ACR_NAME
+   az acr import --source docker.io/weaveworks/kured:1.4.0 -n $ACR_NAME
+   az acr import --source quay.io/k8scsi/csi-node-driver-registrar:v1.2.0 -n $ACR_NAME
+   az acr import --source docker.io/deislabs/secrets-store-csi:v0.0.11 -n $ACR_NAME
+   az acr import --source quay.io/k8scsi/livenessprobe:v1.1.0 -n $ACR_NAME
+   ```
+
 1. Deploy Flux.
 
-   > If you used your own fork of this GitHub repo, consider updating this [`flux.yaml`](./cluster-baseline-settings/flux.yaml) file to include reference to your own repo and change the URL below to point to yours as well. Also, since Flux will begin processing the manifests in [`cluster-baseline-settings/`](./cluster-baseline-settings/) now would be a good time to update the `<replace-with-an-aad-group-object-id-for-this-cluster-role-binding>` placeholder in [`user-facing-cluster-role-aad-group.yaml`](./cluster-baseline-settings/user-facing-cluster-role-aad-group.yaml) with the Object IDs for the Azure AD Group(s) you created for management purposes. If you don't, the manifest will still apply, but AAD integration will not be mapped to your specific AAD configuration.
+   > If you used your own fork of this GitHub repo, update the [`flux.yaml`](./cluster-baseline-settings/flux.yaml) file to include reference to your own repo and change the URL below to point to yours as well. Also, since Flux will begin processing the manifests in [`cluster-baseline-settings/`](./cluster-baseline-settings/) now would be a good time to:
+   >
+   > * update the `<replace-with-an-aad-group-object-id-for-this-cluster-role-binding>` placeholder in [`user-facing-cluster-role-aad-group.yaml`](./cluster-baseline-settings/user-facing-cluster-role-aad-group.yaml) with the Object IDs for the Azure AD group(s) you created for management purposes. If you don't, the manifest will still apply, but AAD integration will not be mapped to your specific AAD configuration.
+   > * Update six `image` manifest references to your container registry instead of the default public container registry. See comment in each file for instructions.
```

Comment:
could we start considering to fork a pre-requisite for all the paths? This way, we could have a first citizen experience on this amazing best practice <3

Ideally in this PR, but I really don't want to block us because of this. A future PR will do as well if that is the team final decision.

Reply:
Understood.  This non-forked flow still "just works" and in the interest of keeping the barrier to entry low, I think we'll have to schedule time in a future sprint to think about reflowing the document.

Reply:
(marking resolved -- if you want to reopen, please do so.  Thanks)